app/assets/stylesheets/custom.scss

```
@@ -73,3 +89,65 @@ body {
 .image-link:hover {
   background-color: transparent;
 }
+
+/*sidebar*/
+aside{
```

```suggestion
aside {
```
sửa lại cho đồng bộ file scss